.github/CODEOWNERS

```diff
@@ -1 +1 @@
-/.github/  @youwol/continuous-integration
+/.github/  @youwol/continuous-integration
```

Comment:
Missing EOL there.
Maybe juste rebase the whole thing to `drop` the only commit modifying `CODEOWNERS`:
```shell
# assuming the checkout branch is `feature/TG-1492` and remote `origin` is the Github repo
git rebase -i origin/main
```